packages/replay/src/integration.ts

```diff
@@ -1,6 +1,5 @@
-import type { BrowserClient, BrowserOptions } from '@sentry/browser';
 import { getCurrentHub } from '@sentry/core';
-import { Integration } from '@sentry/types';
+import { BrowserClientReplayOptions, Integration } from '@sentry/types';
```

Comment:
l/m: Could we keep this as `import type`?

Reply:
[8552e6b](https://github.com/getsentry/sentry-javascript/pull/6508/commits/8552e6bcb459cf204230e2438c941b4682bd9bbe)